dlvm/tools/convert_to_rt.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
perhaps list (and give a brief description of) each tool in the top level readme.